packages/solid/src/__tests__/create_query_resource.test.tsx

```diff
@@ -289,4 +289,51 @@ describe('createQueryResource', () => {
 
     expect(await screen.findByText(2)).toBeInTheDocument();
   });
+
+  test('query update should trigger resource', async () => {
+    const defer = createDefer<any, unknown>();
+    const deferMutation = createDefer<any, unknown>();
+    const controlledQuery = createQuery({ handler: () => defer.promise });
+
+    const mutation = createMutation({
+      handler: async () => 'Mutation success',
+    });
+
+    update(controlledQuery, {
+      on: mutation,
+      by: {
+        success: ({ mutation: { result } }) => ({ result }),
+      },
+    });
+
+    const scope = fork();
+
+    const boundStart = scopeBind(controlledQuery.start, { scope });
+    const boundMutationStart = scopeBind(mutation.start, { scope });
```

Comment:
As I see, it is not used. Should we delete it?

Reply:
Sure, if the tests won't break